package.json

```diff
@@ -206,8 +226,7 @@
         "displayName": "Settings (Organizations): Can view and edit settings",
         "visible": true,
         "subPermissions": [
-          "configuration.all",
-          "organizations-storage.categories.all",
+          "ui-organizations.categories.settings.all",
           "ui-organizations.view"
```

Comment:
Looks like it should be removed. With this permission set, the user can get all data from `ui-organizations.basic.view` but from the permission name, it follows that the user can only view and edit Settings (for now it's only categories).

Reply:
removed